### PR TITLE
fix #8849 feat(nimbus): validate incoming jetstream schemas

### LINKED ISSUE #8849: Consume/use Jetstream schemas during analysis results ingestion
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3501)

### PR DESCRIPTION
**Pull request by mikewilli**
NOTE: depends on https://github.com/mozilla/experimenter/pull/9015 

Because

- we have fancy new nimbus schemas package with jetstream schemas in it

This commit

- adds some validation to the jetstream results ingestion using the new schemas

### CODE REVIEW

**Review by brennie — APPROVED** [review]

File: experimenter/pyproject.toml [hunk 1]
```diff
@@ -66,6 +66,7 @@ pyright = "^1.1.291"
 django-types = "^0.17.0"
 ruff = "^0.0.239"
 mozilla-nimbus-shared = "^2.4.0"
+mozilla-nimbus-schemas = "^2023.6.5"
```
Comment (jaredlockhart) [inline experimenter/pyproject.toml:69]: 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉

**Review by jaredlockhart — CHANGES_REQUESTED** [review]
Yeah run this inside the container and then it should only add the lock changes for just that package 🙏 

But yaaaaayyyy!!!!!  🎉 🎉 🎉 🎉 🎉 🎉

File: experimenter/poetry.lock [hunk 2]
```diff
@@ -1339,6 +1339,20 @@ build = ["blurb", "twine", "wheel"]
...
+description = "Schemas used by Mozilla Nimbus and related projects."
+optional = false
+python-versions = ">=3.10,<4.0"
+files = [
+    {file = "mozilla_nimbus_schemas-2023.6.5-py3-none-any.whl", hash = "sha256:a3ad98e94241ed5586d8cd3cd5bb02e50343264042ef1ef11e0cf23b79737ffb"},
+    {file = "mozilla_nimbus_schemas-2023.6.5.tar.gz", hash = "sha256:ed314284d5aa4ae27aebb44029133b1f76a70389f125638382b87cfcec140736"},
+]
+
+[package.dependencies]
+pydantic = ">=1.10.7,<2.0.0"
```
Comment (jaredlockhart) [inline experimenter/poetry.lock:1354]: Much better 🙏

**Review by jaredlockhart — APPROVED** [review]
Fantastic so awesome 🎉 🎉 🎉 🎉 🎉 🎉 🎉

**Pull request merged**